boot.log

```diff
@@ -0,0 +1,3002 @@
+Apr 01 08:21:13 nixos systemd[1670]: Starting Signal initialization done to GNOME Session Manager...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME accessibility service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME color management service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME date & time service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME maintenance of expirable data service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME keyboard configuration service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME keyboard shortcuts service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME power management service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME printer notifications service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME RFKill support service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME FreeDesktop screensaver service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME file sharing service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME smartcard service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME sound sample caching service...
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME USB protection service...
+Apr 01 08:21:13 nixos .gnome-shell-wr[1759]: Failed to launch ibus-daemon: Failed to execute child process “ibus-daemon” (No such file or directory)
+Apr 01 08:21:13 nixos dbus-daemon[1679]: [session uid=1000 pid=1679] Successfully activated service 'org.gnome.evolution.dataserver.AddressBook10'
+Apr 01 08:21:13 nixos systemd[1670]: Starting GNOME Wacom tablet support service...
+Apr 01 08:21:13 nixos systemd[1670]: Started Evolution address book service.
+Apr 01 08:21:13 nixos gnome-session-binary[1738]: Entering running state
+Apr 01 08:21:13 nixos systemd[1670]: Finished Signal initialization done to GNOME Session Manager.
+Apr 01 08:21:13 nixos systemd[1670]: Started GNOME printer notifications service.
+Apr 01 08:21:13 nixos systemd[1670]: Started GNOME accessibility service.
+Apr 01 08:21:13 nixos systemd[1670]: Started Application launched by gnome-session-binary.
+Apr 01 08:21:13 nixos systemd[1670]: Started Application launched by gnome-session-binary.
+Apr 01 08:21:13 nixos systemd[1670]: Reached target GNOME accessibility target.
+Apr 01 08:21:13 nixos systemd[1670]: Reached target GNOME printer notifications target.
+Apr 01 08:21:13 nixos systemd[1670]: Started GNOME maintenance of expirable data service.
+Apr 01 08:21:13 nixos systemd[1670]: Reached target GNOME maintenance of expirable data target.
+Apr 01 08:21:13 nixos dbus-daemon[1679]: [session uid=1000 pid=1679] Successfully activated service 'org.gnome.Shell.Notifications'
+Apr 01 08:21:13 nixos gnome-session[1738]: gnome-session-binary[1738]: GnomeDesktop-WARNING: Could not create transient scope for PID 2063: GDBus.Error:org.freedesktop.DBus.Error.UnixProcessIdUnknown: Process with ID 2063 does not exist.
```

Comment:
Gnome error 2

Reply:
https://search.brave.com/search?q=Could+not+create+transient+scope+for+PID+2063%3A+GDBus.Error%3Aorg&source=desktop